churchland_pipeline/acquisition.py

```diff
@@ -113,8 +296,7 @@ def make(self, key):
         sessKey = (Session & key).fetch(as_dict=True)[0]
```

Comment:
```suggestion
        sessKey = (Session & key).fetch1()
```
But here `key` should already be the same as `sessKey`.

Reply:
I thought key would only have the same primary attributes as sessKey? Here I wanted to fetch all of the session attributes (to construct the path to the raw data)